### ISSUE TITLE
patient characteristics discrepancies md of seurat object vs table S6

### ISSUE THREAD

**Issue by JoannaG88** [icbi-lab/luca#19]
Hi,

First of all I would like to congratulate the whole team on this fantastic work and for making it available for the community! I was wondering if you could help me figure something out. I was exploring the dataset (downloaded seurat object of the extended LuCa) focusing on the patients with driver mutation, and I noticed that the annotations of the EGFR_mutation of certain patients (e.g. Laughney_Massague_2020_LX666, Laughney_Massague_2020_LX675) in the metadata of the seurat object do not match the info provided in the "Table S6: related to STAR Methods. Patient metadata" ("driver_genes" column). Both patients appear to have EGFR mutation according to the info in the table while in the seurat object metadata (subsetted for CD8 only "using cell_type_major" == "T cell CD8") I assumed that these two sources would have corresponding/matching info, or am I missing something? 

Many thanks in advance!

Best,
Joanna

**Comment by grst** [comment]
Hi @JoannaG88, 

I tried tracing it back and the annotation in the atlas is taken unmodified from the curated `h5ad` object of the Laughney_Massague study. The notebook for curation is not part of this github repo but stored on my old institute's servers. 
Very likely the information comes either from the original single-cell objects of the Laughney_Massague study or from a supplementary table of their paper. 

@riederd, could you possibly share here the `ipynb` of the `Laughney_Massague` dataset? IIRC it's stored in `/data/datasets`. 

The Table S6 was filled manually by the medical colleagues who coauthored the paper. It's possible that they took additional information into account, e.g. what's mentioned in the paper figures or text. It's also entirely possible that a mistake happened during manual curation.

**Comment by riederd** [comment]
Sure, here we go:

[luaghney_make_h5ad.zip](https://github.com/user-attachments/files/17955901/luaghney_make_h5ad.zip)

**Comment by JoannaG88** [comment]
Dear Gregor and Dietmar,

Thanks very much for your very quick reaction and for the info, I really appreciate it. I will have a look into the original dataset, to trace the annotations. Would you suggest to entirely relay on the annotations in the h5ad/Rds files, and take it as ground truth, rather than what is mentioned in the Table S6?

Best,
Joanna

**Comment by grst** [comment]
Hi Joanna, 

I looked in to the notebook shared by Dietmar and it seems I was wrong. The mutation data is not contained there, instead it is actually added to the atlas at a later point from [this table](https://github.com/icbi-lab/luca/blob/5ffb0a4671e9c288b10e73de18d447ee176bef1d/tables/additional_patient_metadata/patient_metadata_corrected.xlsx) with appears to be consistent with Supplementary Table 6. 

The code snippet that adds the annoations to the atlas are
https://github.com/icbi-lab/luca/blob/5ffb0a4671e9c288b10e73de18d447ee176bef1d/analyses/30_annotate_scrnaseq_data/35_export_atlas.py#L112-L133

and 

https://github.com/icbi-lab/luca/blob/5ffb0a4671e9c288b10e73de18d447ee176bef1d/analyses/30_annotate_scrnaseq_data/35_export_atlas.py#L161-L168

I currently don't see why mutations would get lost here. I am investigating whether they are present in the `h5ad` deposited on Zenodo and might have been lost during import on cellxgene. In any case this answers your question that the Table S6 is more accurate and the single point of truth is the Excel sheet on github that was manually curated. My provenance chain ends there.

**Comment by grst** [comment]
ah, damn, it's a typo in the excel sheet: 
`EGRF` instead of `EGFR`

**Comment by JoannaG88** [comment]
Hi Gregor,

Ah no, what a nasty typo! Honest mistake though, can happen and happens all the time. Thanks for looking into this. I thought I saw a typo yesterday when I was checking Table S6, but I did not think it could affect the metadata of the object. Just to let you know, but maybe you already noticed, that the same typo creeped into the He_Fan_2021 study.

Thanks for all your help!

Best,
Joanna

**Comment by JoannaG88** [comment]
Great to know that "patient_metadata_corrected.xlsx" has the correct info, thanks! I had one more question about the condition column in the table, dose it correspond to the origin column in the metadata? According to the table all patients from Laughney_Massague_2020 and He_Fan_2021 are LUAD but in the metadata are annotated as normal_adjacent.

**Comment by JoannaG88** [comment]
> Great to know that "patient_metadata_corrected.xlsx" has the correct info, thanks! I had one more question about the condition column in the table, dose it correspond to the origin column in the metadata? According to the table all patients from Laughney_Massague_2020 and He_Fan_2021 are LUAD but in the metadata are annotated as normal_adjacent.

Hi Gregor! Sorry to bother you again, but any chance you could shed some light onto this one (above)? I would greatly appreciate it.

Many thanks in advance!

Best,
Joanna

**Comment by grst** [comment]
Sorry, seems your previous comment got lost...

* `condition` e.g. LUAD/LUSC is patient-level information, i.e. the patient has a certain disease or not
* `origin`, e. g. `tumor_rimary`/`normal_adjacent` is sample-level information. From a LUAD patient, you could have two biopsies, one being normal_adjacent and one being tumor_primary.

**Comment by JoannaG88** [comment]
Of course! Thank you (and the whole team) once more for such an amazing effort with the Lung Atlas and for all your help now with the annotations. This is a beautiful example of data accessibility and sharing of the science. Thank you!

**Comment by grst** [comment]
> Just to let you know in the Excel file patient "Maier_Merad_2020_NSCLC_571" is annotated as a never smoker (ever_smoker = no) while in the md it has a smoker label (ever_smoker = yes)

What are you referring to here exactly? In cell-x-gene, in the core atlas, Maier_Merad_2020_NSCLC_571 is annotated as `ever_smoker = "no"` which is consistent with the Excel sheet. 

However, it seems that in the extended atlas, in cell-x-gene "Leader_Merad_2021_571" (which should be the same patient) is annotated as `ever_smoker = "yes"`

Is that the inconsistency you meant?

**Comment by JoannaG88** [comment]
Hi Gregor,

I was actually referring to the inconsistency between the curated excel file on github that you mentioned before as a ground truth ("patient_metadata_corrected.xlsx") and the annotations in the extended atlas. Good to know that in the core atlas it is correct. I have downloaded and have been using the extended atlas so far, but I realized that if I am interested in the the T cells the core atlas is sufficient (level of granularity). Would you happen to know if the annotations in the core atlas are perhaps more accurate i.e. correspond to the ones in the "patient_metadata_corrected.xlsx"?

**Comment by grst** [comment]
>  Would you happen to know if the annotations in the core atlas are perhaps more accurate i.e. correspond to the ones in the "patient_metadata_corrected.xlsx"?

"patient_metadata_corrected.xlsx" is *only* for the core atlas. I double-checked and for the studies added to the extended atlas, 
the metadata was added already in the data curation notebooks that I mentioned initially. 

In this case it could be an inconsistency in the original data. Some of the patients from the Leader_Merad study have been published before in a separate paper focussing only on dendritic cells. That's why  the extended atlas *only* contains Leader_Merad but not Maier_Merad. 

I couldn't find the source of the metadata for the Maier_Merad study anymore, but in Supplementary table S1 of Leader_Merad, patient 571 is described as "pack-years > 0", so ever_smoker = "yes" seems to be correct. 

![image](https://github.com/user-attachments/assets/33358f2f-1433-463a-85eb-476566b2748c)

**Comment by JoannaG88** [comment]
Great, thank you for clearing things out!

**Comment by grst** [comment]
Laughney_Massague_2020_LX666 and Laughney_Massague_2020_LX675 as well as the He_Fan patients are now correctly annotated as EGFR-mutated on cell-x-gene, therefore closing this issue. Thanks again for pointing out this inconsistency.
